README.md

```diff
@@ -35,6 +35,12 @@ curl -L https://raw.githubusercontent.com/software-mansion/cairo-coverage/main/s
 
 ## Usage
 
+### `.cairo-coverage-ignore` File
+
+You can create a `.cairo-coverage-ignore` file in the root of your project to specify the files or directories that you
+want to exclude from the coverage report. The file should contain a list of paths to be ignored, with one path per line,
+similar to a `.gitignore` file.
```

Comment:
we could also mention that regexes work there, pretty significant info

Reply:
Added 😄